packages/affiliates/apps/GetSheetsDevminingStatus.js

```diff
@@ -0,0 +1,118 @@
+// This grabs the latest state from dev mining. Outputs it into a format which can be used in multiple ways.
+// Can be used to generate issue in github, or dev mining configuration, or updating dev mining status github.
+// This Requires ENV vars and a user with access to our private dev mining spreadsheet
+// GOOGLE_CREDENTIALS - set up sheet api https://developers.google.com/sheets/api/quickstart/nodejs
+// SHEET_ID - the sheet ID in the url to the dev mining sheet
+// GOOGLE_TOKEN_PATH - oath2 will need to save a token to your drive. Recommend /tmp/google_token
+require("dotenv").config();
+const assert = require("assert");
+const fs = require("fs");
+const { google } = require("googleapis");
+const prompt = require("prompt");
+const web3 = require("web3");
+
+// If modifying these scopes, delete token.json.
+const SCOPES = ["https://www.googleapis.com/auth/spreadsheets.readonly"];
+
+// GDrive Specific: have to use oauth credentials.
+async function auth(credentials) {
+  assert(credentials, "requires credential object");
+  const { client_secret, client_id, redirect_uris } = credentials.installed;
+  return new google.auth.OAuth2(client_id, client_secret, redirect_uris[0]);
+}
+
+function writeObject(path, obj) {
+  assert(path, "requires a path to write object");
+  return fs.writeFileSync(path, JSON.stringify(obj));
+}
+function readObject(path) {
+  assert(path, "requires a path to read object");
+  return JSON.parse(fs.readFileSync(path));
+}
+
+async function getNewToken(oAuth2Client) {
+  assert(oAuth2Client, "requires oath2client");
+  const authUrl = oAuth2Client.generateAuthUrl({
+    access_type: "offline",
+    scope: SCOPES
+  });
+  console.log("Authorize this app by visiting this url:", authUrl);
+  prompt.start();
+  const { code } = await prompt.get(["code"]);
+  console.log("got code:", code);
+  return oAuth2Client.getToken(code);
+}
+
+// This assumes URL in teh format https://etherscan.io/address/0xd81028a6fbaaaf604316f330b20d24bfbfd14478
+// Which could be quite fragile, but works for now.
+function parseAddressFromEtherscan(url) {
+  const addr = url.split("/")[4];
+  return parseAddress(addr);
+}
+// Validates an eth address and upgrades it to checksum address
+function parseAddress(addr) {
+  assert(web3.utils.isAddress(addr), "Invalid Eth address: " + addr);
+  return web3.utils.toChecksumAddress(addr);
+}
+
+// Assumes dev mining status column === 1 for active emp
+function isEnabled(enabled) {
+  return enabled == "1";
```

Comment:
Consider making this a constant `GOOGLE_EMP_IS_ACTIVE` along with all other hardcoded values that are related to values set in the google sheet